utils/table.js

```diff
@@ -26,5 +26,27 @@ module.exports = {
 		`${red(`Critical`)}`,
 		`Per Million`
 	],
+	sortKeys: [
+		`country`,
+		`cases`,
+		`cases-today`,
+		`deaths`,
+		`deaths-today`,
+		`recovered`,
+		`active`,
+		`critical`,
+		`per-million`
+	],
+	sortOrders: [
+		1,
```

Comment:
I like the sort by death count that's set by default. Gives you the best picture. What do you say we make that default instead of sorting by countries?

Reply:
It only sorts if `sort` option is provided. `sortOrders` are to specify `asc`/`desc`

I like the idea of sorting by _cases today_. That gives a better picture of situation than death count.